src/lib/Leaderboards.svelte

```diff
@@ -14,7 +14,7 @@
 
   async function getPlayers() {
     let response = await fetch(
-      `${import.meta.env.VITE_BSABER_API_BASE || 'https://api.bsaber.info'}/top-players`,
+      `${import.meta.env.VITE_BSABER_API_BASE || 'https://api.bsaber.com'}/top-players`,
```

Comment:
Yeah, for now. I did change that back since it breaks the leaderboards. Rorekk mentioned that the backend is hosted separately. So it doesn't move with us simultaneously.